glyrc may fail to find covers if the locale isn't English. Reported under
https://abcde.einval.com/bugzilla/show_bug.cgi?id=83.

--- abcde-2.9.1.orig/abcde	2018-03-10 00:03:50.000000000 +0100
+++ abcde-2.9.1/abcde	2018-05-21 15:13:59.789577384 +0200
@@ -3244,7 +3244,7 @@
 		# use glyrc
 		if [ ! -s "${ABCDETEMPDIR}/$ALBUMARTFILE" ]; then
 			vecho "trying to get cover with glyrc for $ARTISTFILE / $ALBUMFILE" >&2
-			$GLYRC cover --artist "$ARTISTFILE" --album "$ALBUMFILE" --write "${ABCDETEMPDIR}/$ALBUMARTFILE" $GLYRCOPTS
+			env LC_ALL=C $GLYRC cover --artist "$ARTISTFILE" --album "$ALBUMFILE" --write "${ABCDETEMPDIR}/$ALBUMARTFILE" $GLYRCOPTS
 			if [ $? -ne 0 ]; then
 				vecho "could not download cover with glyrc" >&2
 			else
